cql2pgjson-cli/src/test/java/org/z3950/zing/cql/cql2pgjsoncli/TestCLI.java

```diff
@@ -136,8 +101,4 @@ public void testCLIName() throws Exception {
         "select * from instance where to_tsvector('simple', instance.jsonb->>'title') @@ to_tsquery('simple','foo')");
   }
 
-  @Test(expected = QueryValidationException.class)
-  public void testCLIParseException() throws Exception {
-    testCLI("x=foo", null);
-  }
```

Comment:
Please do not remove this test that checks that CQL2PGCLIMain.handleOptions can throw one of the declared exceptions. Using "=" instead of "x=foo" will still throw the exception:

    @Test(expected = QueryValidationException.class)
    public void testCLIParseException() throws Exception {
      testCLI("=", null);
    }